miso-service/src/main/java/uk/ac/bbsrc/tgac/miso/service/impl/DefaultSampleService.java

```diff
@@ -506,6 +506,9 @@ private void applyChanges(Sample target, Sample source) throws IOException {
     target.setDescription(source.getDescription());
     target.setEmpty(source.isEmpty());
     target.setVolume(source.getVolume());
+    if (!isStringBlankOrNull(source.getIdentificationBarcode())) {
```

Comment:
Also, if you can't remove it and someone wants to anyway, they'll just set it to "asdfjkl;"